openedx/core/djangoapps/models/course_details.py

```diff
@@ -43,6 +47,10 @@ def __init__(self, org, course_id, run):
         self.enrollment_start = None
         self.enrollment_end = None
         self.syllabus = None  # a pdf file asset
+        self.title = ""
```

Comment:
These new fields aren't actually hooking up with anything on the server yet, are they? I don't see in this PR where they are persisted or how they are used.